zettarepl/utils/logging.py

```diff
@@ -4,7 +4,24 @@
 
 logger = logging.getLogger(__name__)
 
-__all__ = ["LongStringsFilter"]
+__all__ = ["LoggingLevelContext", "LongStringsFilter"]
+
+
+class LoggingLevelContext:
+    def __init__(self, level):
+        self.level = level
+        self.prev_level = None
+
+    def __enter__(self):
+        self.prev_level = logging.getLogger().level
```

Comment:
To confirm, is it impossible replication tasks can run concurrently?

Reply:
Not yet. I already have a plan on this for multi-threaded program.